help/en/package/jmri/jmrit/display/Tooltips.shtml

```diff
@@ -0,0 +1,117 @@
+      be set to not show a tooltip.  The reverse is also possible where tooltips are disabled for
+      the panel, but individual icons can be enabled.</p>
+
+      <dl>
+        <dt>Panel Editor</dt>
+        <dd>When the panel is in edit mode, use the <strong>Show Tooltips for all items</strong>
+        option to enable or disable tooltips for the panel.
+          <div style="margin-left: 2em">
+            <a href="images/Paneleditor.png"><img src="images/PanelEditor.png" alt="PE Tooltips"
+            width="387" height="370"></a>
+          </div>
+       </dd>
+
+        <dt>Control Panel Editor</dt>
+        <dd>When the panel is in edit mode, select the <strong>Options</strong> menu and use the
+        <strong>Show Tooltips for all items</strong> menu item to enable or disable tooltips for the
+          panel.
+          <div style="margin-left: 2em">
+            <a href="images/tooltips-cpe.png"><img src="images/tooltips-cpe.png" alt="CPE Tooltips"
+            width="347" height="142"></a>
+          </div>
+       </dd>
+
+        <dt >Layout Editor</dt>
+        <dd>The Layout Editor has several options for displaying tooltips.
+          <div style="margin-left: 2em">
+            <a href="images/tooltips-le.png"><img src="images/tooltips-le.png" alt="LE Tooltips"
+            width="469" height="182"></a>
+          </div>
+          Note: Layout Editor does not display tooltips for track components such as turnouts.
+          These are drawn using a set of lines.  They are not icons.
+       </dd>
+      </dl>
+
+      <h3>Tooltip Options</h3>
+
+      <p>The default tooltip shows the user name and system name of the related JMRI item.</p>
+
+      <div style="margin-left: 2em">
+        <a href="images/tooltips-default.png"><img src="images/tooltips-default.png" alt="Default tooltip"
+        width="149" height="52"></a>
+      </div>
+
+      <p>The icon context menu (right click) has an entry for <strong>Tooltips</strong> with 3 menu
+      items.</p>
+
+      <div style="margin-left: 2em">
+        <a href="images/tooltips-menu.png"><img src="images/tooltips-menu.png" alt="Tooltip menu"
+        width="611" height="81"></a>
+      </div>
+
+      <dl>
+        <dt>Show Tooltip</dt>
+        <dd>This is set active when tooltips are enabled for the panel and inactive when tooltips
+        are not enabled for the panel.  Click on the menu item to override the default value for
+        the selected icon.</dd>
+
+        <dt>Set Custom Tooltip...</dt>
+        <dd>The default tooltip displays the user name and system name for the object related to
+        the icon.  The custom tooltip item can be used to display a different value.  Enter the
+        text to be displayed in the window that opens when this item is clicked.  To revert to the
+        default text, remove the text from the window text field.</dd>
+
+        <dt>Prepend Display Name to custom tooltip</dt>
```

Comment:
Respectfully suggest that "prefix" is more common usage (among non-programmers) than "prepend."

Prefix custom tooltip with Display Name
"user name (system name)" will be prefixed to the custom tooltip....

Reply:
> Respectfully suggest that "prefix" is more common usage (among non-programmers) than "prepend."

@dsand47 
What is your view about this? I'm happy to change it if that's better.

Reply:
I can make the prepend/prefix change.

The LE wording also needs some changes.